google_apis.md

```diff
@@ -7,17 +7,30 @@ and use that project for all your environments.
 
 ![google_app_1](images/google_app_1.png)
 
-## Enable Google Maps JS APIs
+* After creating it, make sure that it is selected in the header and enable the Google APIs that you need for this project under `APIs & Services > Library`
 
-* When enabling the Google Maps Javascript API key you can generate one or more keys for your environments.
-What is important, is to restrict the usage of the key to certain domains.
+* When asked, attach the `wg-education CC` as billing account.
 
-![google_app_2](images/google_app_2.png)
+## API key generation
 
-Remember to enable the specific domain for production and both the renuoapp.ch and herokuapp.com domains.
+**Maps JavaScript API**
 
-* When asked, attach the `wg-education CC` as billing account.
+In order to user the Google APIs you will need to generate new credentials. Because we are using only one project per application, but would like to separate the usage in development from the one in production, name the keys like so: `MASTER Maps Public Key` and `DEVELOP Maps Public Key`.
```

Comment:
For example monitoring, in order to know exactly what was used in production and what was used in development

Reply:
I see. This is very good reasoning. 
If that's the case we would need to establish a monitoring process or guidelines. 
I am not sure I want to go in that direction at the moment, especially because most of our projects are very simple and use the APIs just a bit and we never had issues or the need of monitoring until now. I am a bit afraid of (for example in this case) creating 6 keys instead of 2 and maintaining and configuring all of them. I'd tend to simplify but I am very open to different approaches to that. Maybe ask for a third/fourth opinion on that matter, so that we can find a solution that most of us approve. 👍

Reply:
I've talked to Simon and Martin about this issue too. Following points arose after that discussion:

- We already split usage between development and production for analytics, I don't see why we shouldn't split it for these APIs as well.
- For monitoring we have Sentry. It would report that there was an issue and we would look into it if it persists.
- It would be good to have the usage data split between what we use and what the actual users of the application use. This would result in more concrete data.